transports/onion/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "libp2p-onion"
+version = "0.1.0"
```

Comment:
```suggestion
version = "0.1.0-alpha"
```

Like with https://github.com/libp2p/rust-libp2p/pull/2622 and https://github.com/libp2p/rust-libp2p/pull/2289 I suggest releasing this new crate as an `alpha` for now. Objections?

Reply:
Sounds good to me.